packages/terra-collapsible-menu-view/src/terra-dev-site/doc/collapsible-menu-view/ChangeLog.3.doc.mdx

```diff
@@ -0,0 +1,7 @@
+import { Badge } from 'terra-collapsible-menu-view/package.json?dev-site-package';
+
+import ChangeLog from 'terra-terra-collapsible-menu-view/CHANGELOG.md';
```

Comment:
This import has a double `terra` declaration that makes it invalid

Reply:
Fixed here 96a0fe8